docs/installation.rst

```diff
@@ -131,10 +131,12 @@ Using Specific Data Versions
 If you need a specific version of the STPSF data you can download it directly
 
 | Full Versions:
+| `stpsf-data-2.2.0.tar.gz <https://stsci.box.com/shared/static/mjst9j056ibf2uph4gxy8qxmi89tjzwk.gz>`_
 | `stpsf-data-2.1.0.tar.gz <https://stsci.box.com/shared/static/ybp9cupdgxicsn3yj7p9ru9ivuyzcyfw.gz>`_
 | `stpsf-data-2.0.0.tar.gz <https://stsci.box.com/shared/static/i3ui9gfypnhmu5ocec47ooci2ehwf9xf.gz>`_
 
 
 | Min Versions:
+| `minimal-stpsf-data-2.2.0.tar.gz <https://stsci.box.com/shared/static/994a5c7uxc9u9w2li9cwwh440cumbklm.gz>`_
```

Comment:
I thought there were no more min versions?

Reply:
I got that from you after I'd done this. Prior to getting that answer, and because we specifically hadn't changed anything in this data release, I took the 2.1.0 minimal and copied it to be the 2.2.0 minimal. For the next release I'll edit the innerspace and not do this.